packages/core/admin/admin/src/content-manager/components/Wysiwyg/Editor.js

```diff
@@ -29,6 +29,11 @@ const Editor = forwardRef(
     const onChangeRef = useRef(onChange);
 
     useEffect(() => {
+      if (editorRef.current) {
+        // Ensure the editor and its wrapper are cleaned up whenever this view is re-rendered
+        // e.g. in case of re-ordering wysiwyg components in a DynamicZone
+        editorRef.current.toTextArea();
```

Comment:
Im not sure about this change, if placeholder or name are perceived to update then it'll try to focus itself, how does it work when there are many editors rendered at different places? Feels like a source of conflicts

Reply:
When many different editors are in place, there's no conflict as it relates to a specific reference. In my case I actually have about 10 rendered editors within different dynamic blocks.

About your remark on the placeholder and name changing leading to focusing issues:
- placeholder and name are indeed in the dependency array, but I'm under the assumption that these would not dynamically change after page load?
- the `toTextArea` method does not refocus anything. It just cleans up the CodeMirror instance, e.g. removing the wrapper element.